scipy/stats/_distn_infrastructure.py

```diff
@@ -1532,6 +1532,15 @@ def interval(self, confidence=None, *args, **kwds):
             end-points of range that contain ``100 * alpha %`` of the rv's
             possible values.
 
+        Notes
+        -----
+        This is typically implemented as ``ppf([p_tail, 1-p_tail])``, where
+        ``ppf`` is the inverse cumulative distribution function and
+        ``p_tail = (1-confidence)/2``. Suppose ``[c, d]`` is the support of a
+        discrete distribution; then ``ppf([0, 1]) == (c-1, d)``. Therefore,
+        when ``confidence==1`` and the distribution is discrete, the left end
+        of the interval will be beyond the support of the distribution.
```

Comment:
add something like

In discrete distributions the interval will limit the probability in each tail to to be less or equal to p_tail.

Reply:
and will in general be strictly less